zephyr/zephyr_reporter.py

```diff
@@ -119,7 +119,7 @@ def update_zephyr(test_cases_list):
         proj = jira.project(z["project"])
         verid = get_jira_release_id(z['release'], jira, proj)
         cycleName = z["cycle"]
-        cycleName = cycleName + "_" + str((datetime.datetime.now()).strftime("%Y%m%d%H%M%S"))
+        cycleName = cycleName.replace("date", str((datetime.datetime.now()).strftime("%Y%m%d")))
```

Comment:
In the future I think we just need to change the config file from JSON to a python file and we can encode whatever logic we want in python for each config we use...

For example

```
import datetime

config = {
	"report": "output.xml",
	"metafile": "zephyr/boardfarm_tc_meta_file.csv",
	"project": "PROJECT",
	"release": "RELEASE",
	"environment": "Test",
	"cycle": "CYCLE_%s_ABC" % datetime.datetime.now()).strftime("%Y%m%d%H%M%S"),
	"build": "DemoBuild",
	"jira_url": "JIRA_URL",
	"user": "USER",
	"passwd": "PASS",
	"updateautomationstatus": true,
	"status_codes": {
		"UNEXECUTED": -1,
		"PASS": 1,
		"FAIL": 2,
		"NOT TESTED": 7
	}
}
```

Reply:
If a change is required can you change the date replace from 'date' to '<DATE>'

Reply:
```
<DATE>
```

Reply:
Not only date need to modify, also "release" and "cycle" need to change.
But for now I modify those manually.
Can you help me how can I pass value from boardfarm to zephyr_reporter?
I can snmpget fw version and modify to release and cycle format, but can't pass those parameters now.

Reply:
We discusses this privately. Please use the info_for_remote dict and self.logged in the tests..